### PR TITLE
Feature/array1

### PR DESCRIPTION
**Pull request by adushyn**

### CODE REVIEW

File: src/arrays/practice/1.js [hunk 1]
```diff
@@ -1,9 +1,35 @@
-function arrayManipulations() {
-  // your code here
-  let arr = [1,2,3];
-  arr.filter((elem, i) => {
-      console.log(elem);
-  });
+function arr1(arr) {
+    let LastItem = arr[arr.length - 1];
```
Comment (ODushyn) [inline src/arrays/practice/1.js]: Keep variables in camel case. LastItem -> lastItem

File: src/arrays/practice/1.js [hunk 2]
```diff
@@ -1,9 +1,35 @@
-function arrayManipulations() {
-  // your code here
-  let arr = [1,2,3];
-  arr.filter((elem, i) => {
-      console.log(elem);
-  });
+function arr1(arr) {
```
Comment (ODushyn) [inline src/arrays/practice/1.js:1]: Rename function name to something more verbose

File: src/arrays/practice/1.js [hunk 3]
```diff
@@ -1,9 +1,35 @@
...
+function arr3(arr, n) {
+    arr.push(n);
+    return arr;
+}
+
+console.log(arr3([1, 33, 5, 7, 8], 9));
+console.log(arr3([1, 33, 5, 7, 8], 10));
+
+
+function arr4(arr) {
```
Comment (ODushyn) [inline src/arrays/practice/1.js:24]: Make function dynamic. Function should take arrSong as input parametr

File: src/arrays/practice/1.js (hunk 1 above)
Comment (adushyn) [reply src/arrays/practice/1.js]: done

File: src/arrays/practice/1.js (hunk 2 above)
Comment (adushyn) [reply src/arrays/practice/1.js:1]: It was just for me

File: src/arrays/practice/1.js (hunk 3 above)
Comment (adushyn) [reply src/arrays/practice/1.js:24]: problem solved

File: src/arrays/practice/1.js [hunk 4]
```diff
@@ -1,5 +1,5 @@
 function arr1(arr) {
-    let LastItem = arr[arr.length - 1];
+    let lastItem = arr[arr.length - 1];
     return LastItem;
```
Comment (ODushyn) [inline src/arrays/practice/1.js]: Now your program does not work at all. 
Change `Lastitem` -> `lastItem`

**Pull request merged**